Lesson 4/dmytro.bondar/script_nginx.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#check if running as root
+id=$(id -u)
+if [ "$id" -ne "0" ]
+
+then
+	echo "Please run as root" 
+	exit 0
+
+fi
+# Task 1
+if 
+	dpkg --list | grep nginx #check if nginx installed
+# Task 1.1
+then 
+	v=$(nginx -v 2>&1)
+	apt purge nginx --yes; #completely remove nginx
+	echo "$v" "was removed successfully"
+#Task 1.2
+else
+ 	echo "no nginx installed" 
+fi
+#Task 2
+echo "Installing nginx"
+apt install curl gnupg2 ca-certificates lsb-release --yes
```

Comment:
Package gnupg2 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package curl
E: Package 'gnupg2' has no installation candidate
E: Unable to locate package ca-certificates
E: Unable to locate package lsb-release